interface/therapy_groups/therapy_groups_views/addGroup.php

```diff
@@ -0,0 +1,172 @@
+                        <div class="col-md-6 col-sm-6 attach-input">
+                            <span class="bold"><?php echo xlt('Starting date'); ?>:</span>
+                        </div>
+                        <div class="col-md-6 col-sm-6">
+                            <input type="text" name="group_start_date" class="full-width datepicker"  value="<?php echo attr($groupData['group_start_date']);?>">
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row group-row">
+                <div class="col-md-4">
+                    <span class="bold"><?php echo xlt('Type of group'); ?>:</span>
+                    <label class="radio-inline radio-pos">
+                        <input type="radio" value="1" name="group_type" <?php echo is_null($groupData['group_type']) || $groupData['group_type'] == '1' ? 'checked' : '';?>><?php echo xlt('Closed'); ?>
+                    </label>
+                    <label class="radio-inline radio-pos">
+                        <input type="radio" value="2" name="group_type"  <?php echo $groupData['group_type'] == '2' ? 'checked' : '';?>><?php echo xlt('Open'); ?>
+                    </label>
+                    <label class="radio-inline radio-pos">
+                        <input type="radio" value="3" name="group_type"  <?php echo  $groupData['group_type'] == '3' ? 'checked' : '';?>><?php echo xlt('Train'); ?>
+                    </label>
+                </div>
+                <div class="col-md-4">
+                    <span class="bold"><?php echo xlt('Obligatory participation'); ?>:</span>
+                    <label class="radio-inline radio-pos">
+                        <input type="radio" value="1" name="group_participation" <?php echo is_null($groupData['group_participation']) || $groupData['group_participation'] == '1' ? 'checked' : '';?>><?php echo xlt('Mandatory'); ?>
+                    </label>
+                    <label class="radio-inline radio-pos">
+                        <input type="radio" value="2" name="group_participation" <?php echo $groupData['group_participation'] == '2' ? 'checked' : '';?>><?php echo xlt('Optional'); ?>
+                    </label>
+                </div>
+                <div class="col-md-4">
+                    <div class="row">
+                        <div class="col-md-6 col-sm-6 attach-input">
+                            <span class="bold"><?php echo xlt('Status'); ?>:</span>
+                        </div>
+                        <div class="col-md-6 col-sm-6">
+                            <select name="group_status" class="full-width"  value="<?php echo attr($groupData['group_status']);?>">
+                                <?php foreach($statuses as $key => $status): ?>
+                                    <option value="<?php echo attr($key);?>" <?php echo $key == $groupData['group_status'] ? 'selected' : ''; ?>><?php echo xlt($status); ?></option>
```

Comment:
If this is the lists that you created above and the $key are the numbers, then you can translate title in the actual list with xl().
I'll show an example above in the list